packages/bot-web-ui/src/components/quick-strategy/quick-strategy.scss

```diff
@@ -54,6 +70,20 @@
                 justify-content: center;
             }
             #{$quick-strategy}__input {
+                &-field {
+                    &:focus,
+                    &:disabled,
+                    &:not(:focus):not([value='']) {
+                        outline: none;
+
+                        & ~ label {
+                            transform: translate(0, -1.8rem) scale(0.875);
+                        }
+                    }
+                }
+                &-label {
+                    font-size: var(--text-size-xxs);
```

Comment:
Why didn't we use the `Text` component for this?

Reply:
so we will need  to rewrite all our inputs everywhere to use <Text> and its props. Now everywhere styles are used and I need to change font size only in 1 small place. So it is much easier to just change styles in 1 place

Reply:
It's better to repeat `Text` since we future proof it that way so if we ever need to change one text font size or weight, we don't have to give it another class, that's one of the purposes of the `Text` component, since it's just font-size here and it's the same for all, it's okay. But please use the `Text` component over classes everywhere you can.